src/pages/communities.astro

```diff
@@ -12,7 +12,15 @@ import CardCommunity from '../components/CardCommunity.astro';
     <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8">
       {
         communities.map((community:Community) => (
-          <CardCommunity {...community} />
+          <CommunityCard
+            title={community.name}
+            description={community.description}
+            tags={community.tags}
+            githubUrl="https://github.com"
+            linkedinUrl="https://linkedin.com"
+            instagramUrl="https://instagram.com"
+            logoUrl={community.image}
```

Comment:
mmm I'm not sure about that. I think my approach is more clear because it is explicit about what props are passed. I know it is harder to maintain. @IvanM9 what do you think about this?

Reply:
> mmm I'm not sure about that. I think my approach is more clear because it is explicit about what props are passed. I know it is harder to maintain. @IvanM9 what do you think about this?

Sure, but actually `Community` interface can help you to visualize what are you sending as props to `CommunityCard`, and what should you receive as props inside component :)